src/visualizations/EventList.vue

```diff
@@ -25,6 +25,13 @@ div
               icon(name="tags")
               // TODO: Add some kind of highlighting to key
               | {{ key }}: {{ val }}
+            span.field.float-right
+              b-btn(v-b-modal="'edit-modal-' + event.id", variant="outline-dark" size="sm")
+                | Edit
+
+          // TODO: Refactor modal into event-editor component?
```

Comment:
Good idea!

Reply:
Wasn't sure how to do this at first glance however, due to how modals are enabled (see the v-b-modal="blabla" a couple of lines above). So post-poned for now.